web/src/components/gnd-feature-type-editor/gnd-form-editor.js

```diff
@@ -48,6 +53,23 @@ const styles = (theme) => ({
   },
 });
 
+const DragHandle = sortableHandle(() => <span>::</span>);
+
+const SortableItem = sortableElement(({element, onChange}) => (
```

Comment:
You can try setting the z-index on a helper as in https://github.com/clauderic/react-sortable-hoc/issues/87#issuecomment-405911683 to make sure the item stays above the modal while dragging.